addons/website/static/src/scss/website.scss

```diff
@@ -201,6 +201,33 @@ body {
         @include body-image-bg-style();
     }
 
+    > main {
+        > .page_breadcrumb {
+            a {
+                color: inherit !important;
+            }
+            > section {
+                @include o-apply-colors('breadcrumb');
+                @include o-apply-colors('breadcrumb-custom');
+                @include o-add-gradient('breadcrumb-gradient');
+            }
```

Comment:
The applied background is not shown when I use the colorpicker... was this tested? I mean, it's the main colorpicker of the feature :grimacing: 
![image](https://github.com/user-attachments/assets/33885be5-e836-4855-945f-13e0cf10e4cd)

Also if you play a bit more, you'll find more bugs:
![image](https://github.com/user-attachments/assets/887fc65a-9ff1-4a6f-948f-4fb905843281)

Somehow I have random colors (probably the ones I used while in "Over the content") in "Regular" mode

cc @lebl-odoo